ci/travis/install.sh

```diff
@@ -21,3 +21,7 @@ conda update -q conda
 conda install pip conda-build anaconda-client
 # Useful for debugging any issues with conda
 conda info -a
+
+# Add channel to get boost libraries. Probably want to "self-host",
+# maybe embedding their recipes as a git module?
+conda config --add channels menpo
```

Comment:
Note that this is still suboptimal: to install diamond our users would have to add two channels to their configs, which I think hampers usability. To solve that, we need to either get it working with the default boost, or host the menpo recipes and keep them in the biocore channel (as suggested by Patrick).